locales/en/cie/noNfcConnection.md

```diff
@@ -0,0 +1,3 @@
+It appears that your smartphone has not detected a valid card.  
+Rest it on the back of the device and try again. When the phone recognizes the card, it will still need a few seconds to read the data.  
```

Comment:
```suggestion
Hold the card as close as possible to the back of the device. When the phone recognizes the card, it will still need a few seconds to read the data.  
```